osquery/utils/aws/aws_util.cpp

```diff
@@ -228,6 +229,20 @@ OsquerySTSAWSCredentialsProvider::GetAWSCredentials() {
   // Grab system time in seconds-since-epoch for token expiration checks.
   size_t current_time = osquery::getUnixTime();
 
+  // config provides STS creds that includes the token
+  if (!FLAGS_aws_session_token.empty()) {
+    if (access_key_id_.empty()) {
+      initAwsSdk();
+      access_key_id_ = FLAGS_aws_access_key_id;
+      secret_access_key_ = FLAGS_aws_secret_access_key;
+      session_token_ = FLAGS_aws_session_token;
+      VLOG(1) << "Using provided aws_session_token for id starting with :"
```

Comment:
Ah, sorry there's another nitpick here, `with: `, right now there's a space between `with` and `:` and not space between the `:` and the first-8-chars of the key.